chef/cookbooks/pacemaker/spec/fixtures/keystone_primitive.rb

```diff
@@ -7,7 +7,7 @@ module Config
       KEYSTONE_PRIMITIVE = ::Pacemaker::Resource::Primitive.new('keystone')
       KEYSTONE_PRIMITIVE.agent = "ocf:openstack:keystone"
       KEYSTONE_PRIMITIVE.params = [
-        [ "os_password",    "admin$pw"               ],
+        [ "os_password",    "ad\"min$pass'word"      ],
```

Comment:
Would it make sense to also put some `&` and `%` in there as these are the recent characters that were failing?